CHANGELOG.md

```diff
@@ -7,7 +7,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/atomist-rugs/spring-boot-rest-service/compare/0.6.0...HEAD
+[Unreleased]: https://github.com/atomist-rugs/spring-boot-rest-service/compare/0.7.1...HEAD
+
+Changes include:
+
+- Migrated generator to TypeScript and Rug 0.12.0.
+
+## [0.7.1] - 2017-01-03
+
+[0.7.0]: https://github.com/atomist-rugs/spring-boot-rest-service/compare/0.6.0...0.7.1
```

Comment:
What version are we releasing? These numbers are all over the place. Is 0.6.0 that last release? If so, this should be 0.7.0.